src/RealtimeEffectPanel.cpp

```diff
@@ -866,9 +944,12 @@ class RealtimeEffectListWindow : public wxScrolledWindow
       mAddEffectHint = addEffectHint;
 
       //TODO: set link
-      auto addEffectTutorialLink = safenew ThemedWindowWrapper<wxHyperlinkCtrl>(this, wxID_ANY, "x", "https://www.audacityteam.org");
+      auto addEffectTutorialLink = safenew ThemedWindowWrapper<wxHyperlinkCtrl>(this, wxID_ANY, _("Watch video"), "https://www.audacityteam.org");
```

Comment:
I see that the template that became unused earlier now is reused.

I see _ again but the template adds the handler for preference changes that updates the title.

The handler for preference change in that template, and also in the `ThemedButtonWrapper`, should also update the window name with the title, for the screen reader.